### ISSUE TITLE
Cant get filter to work on tile 

### ISSUE THREAD

**Issue by JacobHonig** [taarskog/crm-powerbi-viewer#48]
Hi 

I have created a Power BI report and pined some tiles to a dashboard. I can see the dashboad and tiles in the PowerBIViewer configuration (but not any reports). I have copied the tile configuation and created a web resource on the form with the following 

type=tile&id=b4b059db-f385-4093-80ef-de4ce52c0bb6&dashboardId=e2b598ce-41d4-4cfd-a9c9-7d4fcc00c71f

![image](https://user-images.githubusercontent.com/44808035/48071368-752fff00-e1da-11e8-9de9-06c4bd661eb3.png)

The tile **with data** is shown on the form but not filtered, so i add 

&filterFn=Samples.Filters.filterOnAccount

![image](https://user-images.githubusercontent.com/44808035/48070004-29c82180-e1d7-11e8-888b-5b748f04e7db.png)

The filterOnAccount script looks like this

```
window.Samples = window.Samples || {};
Samples.Filters = Samples.Filters || {};

Samples.Filters.filterOnAccount = function () {

    var accid = window.parent.Xrm.Page.data.entity.getId();
    accid = accid.replace('{','').replace('}','');

    return [
        {
            $schema: "http://powerbi.com/product/schema#basic",
            target: {
                table: "contacts",
                column: "_parentcustomerid_value"
            },
            operator: "Eq",
            values: ["7b8fb6d5-f5e0-e811-a958-000d3aba014b"]
        }
    ];
}
```
I steped through the function in the debugger, and the script is running and the filter is returned

But the tile on the form is not filtered

![image](https://user-images.githubusercontent.com/44808035/48072372-a3aed980-e1dc-11e8-98cd-199c78fb06aa.png)

**Comment by taarskog** [comment]
Reports should be shown. I would look into why you cannot see them as filtering dashboard tiles are not supported in Power BI.

When you have fixed the report issue you can filter and show a "tile" by adding [single visuals](http://crm-powerbi-viewer.heiigjen.com/pages/add-view-to-dashboard.html#single-visual).

**Comment by JacobHonig** [comment]
Thanks

Have you any suggestions to why the report is not showing in the solution?

I have created the report (1) in Power BI, and pined the tile in the report to a new dashboard (2). 

![image](https://user-images.githubusercontent.com/44808035/48120209-cc34e300-e271-11e8-9c3f-704f3d65e9fe.png)

But only the new dashboard is showing in the solution.

![image](https://user-images.githubusercontent.com/44808035/48120271-ff777200-e271-11e8-9a2a-6eb6d138a795.png)

**Comment by JacobHonig** [comment]
I solved why the report is not showing! The reports was not showing because some of the permissions was not set in AAD. 

I used the new guide that created the permissions automaticaly. But not all of the permissions pictured in the olg guid was selected. Ex. "View all Reports (preview)" was not selected. 

![image](https://user-images.githubusercontent.com/44808035/48121225-a52be080-e274-11e8-9d70-a26931a3be26.png)

When i selected them, the report is showing in the solution

![image](https://user-images.githubusercontent.com/44808035/48121324-e9b77c00-e274-11e8-9471-f21d131cf4ff.png)

Now i will try filter on the report.
